package.json

```diff
@@ -13,6 +13,7 @@
     ]
   },
   "scripts": {
+    "prebuild": "yarn workspace @kablamo/kerosene run build",
```

Comment:
This seems clunky, and hopefully theres a better way to do it. Let's leave it for now, I've made #21 to cover this.

Would changing `build` to `yarn prebuild && yarn workspaces run build` work? We could then load up prebuild with ordered build dependencies if we need.

Reply:
`prebuild` is run implicitly when `build` is run. `@kablamo/kerosene-ui` actually doesn't depend on the local copy of `@kablamo/kerosene`, but it seems like the TS typings do depend on the local version (which get deleted when the `clean` setp is run). From what I can tell, there doesn't seem to be a way to control the order that packages build in.